app/scenes/more_dms/more_dms.js

```diff
@@ -0,0 +1,196 @@
+            return (
+                <TouchableOpacity
+                    style={{flex: 1, paddingHorizontal: 15, justifyContent: 'center'}}
+                    onPress={() => emitter('close')}
+                >
+                    <FormattedText
+                        id='more_direct_channels.close'
+                        defaultMessage='Close'
+                        style={{color: theme.sidebarHeaderTextColor}}
+                    />
+                </TouchableOpacity>
+            );
+        }
+    };
+
+    constructor(props) {
+        super(props);
+
+        this.searchTimeoutId = 0;
+
+        this.state = {
+            profiles: [],
+            page: 0,
+            next: true,
+            searching: false
+        };
+    }
+
+    componentWillMount() {
+        this.props.subscribeToHeaderEvent('close', this.props.actions.goBack);
+    }
+
+    componentWillReceiveProps(nextProps) {
+        const {requestStatus} = this.props;
+        if (requestStatus.status === RequestStatus.STARTED &&
+            nextProps.requestStatus.status === RequestStatus.SUCCESS) {
+            const {page} = this.state;
+            const profiles = nextProps.profiles.splice(0, (page + 1) * Constants.PROFILE_CHUNK_SIZE);
+            this.setState({profiles});
+        } else if (this.state.searching &&
+            nextProps.searchRequest.status === RequestStatus.SUCCESS) {
+            this.setState({profiles: nextProps.search});
+        }
+    }
+
+    componentDidMount() {
+        InteractionManager.runAfterInteractions(() => {
+            this.props.actions.getProfiles(0);
+        });
+    }
+
+    componentWillUnmount() {
+        this.props.unsubscribeFromHeaderEvent('close');
+    }
+
+    searchProfiles = (event) => {
+        const term = event.nativeEvent.text;
+
+        if (term) {
+            this.setState({searching: true});
+            clearTimeout(this.searchTimeoutId);
+
+            this.searchTimeoutId = setTimeout(() => {
+                this.props.actions.searchProfiles(term);
+            }, Constants.SEARCH_TIMEOUT_MILLISECONDS);
+        } else {
+            this.cancelSearch();
+        }
+    };
+
+    cancelSearch = () => {
+        this.setState({
+            searching: false,
+            page: 0,
+            profiles: []
+        });
+    };
+
+    loadMoreProfiles = () => {
+        let {page} = this.state;
+        if (this.props.requestStatus.status !== RequestStatus.STARTED && this.state.next && !this.state.searching) {
+            page = page + 1;
+            this.props.actions.getProfiles(page, Constants.PROFILE_CHUNK_SIZE).
+            then((data) => {
+                if (Object.keys(data).length) {
+                    this.setState({
+                        page
+                    });
+                } else {
+                    this.setState({next: false});
+                }
+            });
+        }
+    };
+
+    onSelectMember = (id) => {
+        this.searchBar.blur();
```

Comment:
The thing is that the keyboard does not belong to the `ListView` so the listview it belongs to the `searchBar`